solutions/devsprint-bruno-faganello-1/FinanceApp/New Group/FinanceApp+Double.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  FinanceApp+Double.swift
+//  FinanceApp
+//
+//  Created by Victor Catão on 17/03/22.
+//
+
+import Foundation
+
+extension Double {
```

Comment:
[Suggestion]
Create a Folder Called Extension and Rename File to Double+Extension